projects/core/src/lib/components/edit/edit-dialog/edit-dialog.component.ts

```diff
@@ -123,6 +123,32 @@ export class EditDialogComponent implements OnInit {
       });
   }
 
+  public delete(layerId: string, currentFeature: FeatureWithMetadataModel) {
+    this.store$.select(selectViewerId)
+      .pipe(
+        take(1),
+        concatMap(viewerId => {
+          if (!viewerId) {
+            return of(null);
+          }
+          return this.editFeatureService.deleteFeature$( viewerId, layerId, {
+            __fid: currentFeature.feature.__fid,
+            attributes: currentFeature.feature.attributes,
+          });
+        }),
+      )
+      .subscribe(succes => {
+        if (succes) {
+          this.closeDialog();
+          this.mapService.getLayerManager$()
```

Comment:
Maybe we can add this as 'shortcut' inside the MapService so its easier to use, something like 
`mapService.refreshLayer(layerId)` where the MapService then executes this code. This is probably something we do more often in the future.

Reply:
✅